NitroxModel/Discovery/Models/GameLibraries.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+
+namespace NitroxModel.Discovery.Models;
+
+[Flags]
+public enum GameLibraries
+{
+    UNKNOWN = 0,
+
+    /// <summary>
+    /// Steam registry finder
+    /// </summary>
+    STEAM = 1,
+
+    /// <summary>
+    /// Epic games finder
+    /// </summary>
+    EPIC = 2,
+
+    /// <summary>
+    /// Microsoft store finder
+    /// </summary>
+    MICROSOFT = 3,
+
+    /// <summary>
+    /// Discord game store finder
+    /// </summary>
+    DISCORD = 4,
+
+    /// <summary>
+    /// Local config value finder
+    /// </summary>
+    CONFIG = 10,
+
+    /// <summary>
+    /// Environment variable value finder
+    /// </summary>
+    ENVIRONMENT = 11,
+
+    /// <summary>
+    /// Current directory finder
+    /// </summary>
+    CURRENT_DIRECTORY = 12,
```

Comment:
See https://learn.microsoft.com/en-us/dotnet/api/system.flagsattribute
```suggestion
    STEAM = 1,

    /// <summary>
    /// Epic games finder
    /// </summary>
    EPIC = 2,

    /// <summary>
    /// Microsoft store finder
    /// </summary>
    MICROSOFT = 4,

    /// <summary>
    /// Discord game store finder
    /// </summary>
    DISCORD = 8,

    /// <summary>
    /// Local config value finder
    /// </summary>
    CONFIG = 16,

    /// <summary>
    /// Environment variable value finder
    /// </summary>
    ENVIRONMENT = 32,

    /// <summary>
    /// Current directory finder
    /// </summary>
    CURRENT_DIRECTORY = 64,
```

Reply:
Can also use binary notation if preferred: 
 - `0b100` which is 4
 - `0b1000` which is 8
 - ..